Input dropout (#5636)
RasaHQ/rasa: changelog/5636.bugfix.rst

> Add dropout before FFNN after sparse for dense layer in ``DIETClassifier``

Reviewer: ```suggestion
Add dropout between ``FFNN`` and ``SparseForDense`` layers in ``DIETClassifier``
```
